src/Features/LanguageServer/Protocol/Handler/Completion/CompletionResolveHandler.cs

```diff
@@ -38,9 +38,13 @@ public override async Task<LSP.CompletionItem> HandleRequestAsync(LSP.Completion
             {
                 data = (CompletionResolveData)completionItem.Data;
             }
+            else if (completionItem.Data is JToken token)
```

Comment:
Yeah, I'll debug it and see. @allisonchou do you know why the completion handlers have this fallback? Was it an early LSP client bug?

Reply:
I can't even get breakpoints here to hit ¯\\\_(ツ)_/¯

Reply:
I'm not sure why the completion handler has this fallback, it looks like it's been present since the inception of the file. Can we just remove this check and do what the CodeActionsResolveHandler does [here](https://github.com/dotnet/roslyn/blob/master/src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionResolveHandler.cs#L56)?

Reply:
Couldn't get any breakpoints to hit in completion resolve, but for code action resolve you're right its always just json data (anything else doesn't make sense anyway), but I worked out what this is for: Its for the tests, they directly assign a CompletionResolveData object. Should have realised a bit earlier 🤦‍♂️

Reply:
I think we may be able to still get rid of the check if we fix up the tests. I ran into a similar problem while working on code actions, and had to modify the test code so that we [create a JToken](https://github.com/dotnet/roslyn/blob/1ff98f58061daf5065ec5cc32186792e6604b03e/src/Features/LanguageServer/ProtocolUnitTests/CodeActions/CodeActionsTests.cs#L222) instead of a CompletionResolveData object.

Reply:
Yep, I fixed the tests :)